peer/peer.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	// MaxProtocolVersion is the max protocol version the peer supports.
-	MaxProtocolVersion = wire.MaxBlockSizeVersion
+	MaxProtocolVersion = wire.FeeFilterVersion
```

Comment:
Not sure what I can really add anything that the comment doesn't already say. The maximum supported protocol version that the peer package is capable of is now that version.

Peers negotiate the highest shared protocol version between them during the initial handshake and then speak that version to one another.